x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/execution/search/SourceGeneratorTests.java

```diff
@@ -95,13 +94,13 @@ public void testSortFieldSpecified() {
         FieldSortBuilder sortField = fieldSort("test").unmappedType("keyword");
         
         QueryContainer container = new QueryContainer()
-                .sort(new AttributeSort(new FieldAttribute(new Location(1, 1), "test", new KeywordEsField("test")), Direction.ASC,
+                .sort(new AttributeSort(new FieldAttribute(Source.EMPTY, "test", new KeywordEsField("test")), Direction.ASC,
```

Comment:
Here you replaced `new Location(1,1)` with `Source.EMPTY`, but [here](https://github.com/elastic/elasticsearch/pull/36912/files#diff-c555d1468ca62f7fbc3f7ec9eb513e79R24) you had a different approach.